Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -251,40 +253,68 @@ void HotkeyScheduler::Run()
 
       // Graphics
       if (IsHotkey(HK_INCREASE_IR))
+      {
```

Comment:
Not directly related to your PR, but ugh. Global variable, magic numbers without comments. Couldn't be worse

Reply:
Already complained about it in IRC. It's on my "to refactor" list.